docs/victoriametrics/stream-aggregation.md

```diff
@@ -1124,27 +1124,67 @@ See also [aggregation outputs](#aggregation-outputs).
 
 ## Dropping unneeded labels
 
-If you need dropping some labels from input samples before [input relabeling](#relabeling), [de-duplication](#deduplication)
-and [stream aggregation](#aggregation-outputs), then the following options exist:
-
-- To specify comma-separated list of label names to drop in `-streamAggr.dropInputLabels` command-line flag
-  or via `-remoteWrite.streamAggr.dropInputLabels` individually per each `-remoteWrite.url`.
-  For example, `-streamAggr.dropInputLabels=replica,az` instructs to drop `replica` and `az` labels from input samples
-  before applying de-duplication and stream aggregation.
-
-- To specify `drop_input_labels` list with the labels to drop in [stream aggregation config](#stream-aggregation-config).
-  For example, the following config drops `replica` label from input samples with the name `process_resident_memory_bytes`
-  before calculating the average over one minute:
-
-  ```yaml
-  - match: process_resident_memory_bytes
-    interval: 1m
-    drop_input_labels: [replica]
-    outputs: [avg]
-    keep_metric_names: true
-  ```
+To optimize performance and reduce [churn rate](https://docs.victoriametrics.com/guides/understand-your-setup-size/#churn-rate), 
+certain labels can be dropped from incoming samples before [input relabeling](#relabeling), 
+[de-duplication](#deduplication) and [stream aggregation](#aggregation-outputs) are applied. 
+There are several ways to drop labels, which can be used independently or together.
+
+**Global Label Drop** is configured using the `-streamAggr.dropInputLabels` flag. 
+This approach drops specified labels for all remote write targets globally. 
+The flag can be set multiple times, and each instance may contain a comma-separated list of labels. 
+All specified labels across flags are merged and dropped from all remote write destinations.
+
+The following example drops the labels `replica`, `az`, `instance`, and `job` 
+before de-duplication and aggregation for both `foo` and `bar` remote write targets:
+
+```bash
+vmagent \
+  -remoteWrite.url="http://foo/api/v1/write" \
+  -remoteWrite.url="http://bar/api/v1/write" \
+  -streamAggr.config="aggr.yaml" \
+  -streamAggr.dropInputLabels="replica,az" \
+  -streamAggr.dropInputLabels="instance,job"
```

Comment:
I find this example super confusing. I know it works like you described. But my initial understanding of it was that ` -streamAggr.dropInputLabels` is indiviudally applied to corresponding `-remoteWrite.url`. Which made me doubting why ` -streamAggr.dropInputLabels` is called global :)
I would suggest keeping only one usage of this flag to emphasize its "global" appliance:
```
vmagent \
  -remoteWrite.url="http://foo/api/v1/write" \
  -remoteWrite.url="http://bar/api/v1/write" \
  -streamAggr.config="aggr.yaml" \
  -streamAggr.dropInputLabels="replica,az,instance,job"
```